themes/aframe/source/css/_settings.styl

```diff
@@ -17,6 +17,7 @@ $code-font-size = 0.8rem
 
 // Colors
 $dark   = #2c3e50
+$fair = lighten(#2c3e50, 20%)
```

Comment:
how's this look?

![image](https://cloud.githubusercontent.com/assets/203725/15689245/e5f48fa4-2732-11e6-845e-12ab572f9027.png)

related to issue #259